hardware/powersupply.md

```diff
@@ -22,7 +22,7 @@ Power from the micro:bit can be provided by the 3V and GND pins to small externa
 
 It's is important to stay within the design parameters of the board:
 
-- When powered from USB, the on-board interface chip (KL26<span class="v1">V1</span>/KL27<span class="v2">V2</span>) uses its on-chip regulator to provide power, and this chip is rated at a maximum of 120mA.
+- When powered from USB, the on-board interface chip on <span class="v1">V1</span> (KL26) uses its on-chip regulator to provide power, and this chip is rated at a maximum of 120mA.
 
 - The on-board current budget will vary depending on the use of the display, the Bluetooth, microphone, speaker and other peripherals. You should allow a worst case budget of 30mA for when all on-board peripherals are in use, leaving <span class="v1">V1</span>90mA/<span class="v2">V2</span>270mA for circuits plugged into the edge connector.
```

Comment:
```suggestion
- The on-board current budget will vary depending on the use of the display, the Bluetooth, microphone, speaker and other peripherals. You should allow a worst case budget of 30mA for when all on-board peripherals are in use, leaving 90mA <span class="v1">V1</span> / 270mA <span class="v2">V2</span> for circuits plugged into the edge connector.
```

Matches the format elsewhere.

Reply:
added in force push https://github.com/microbit-foundation/dev-docs/compare/b96c9a28afef09f7aee657bd4f43ec641afa0d3a..400811505ff84b6d6402dc6ccbd92c1283666b27